code/web/src/modules/StyleSurvey/StyleSurvey.js

```diff
@@ -85,33 +85,113 @@ class StyleSurvey extends Component {
 
         this.state = {
             currentAnswer: null,
-            userAnswers : [],
-            images: ['https://images.unsplash.com/photo-1525490829609-d166ddb58678?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MXx8fGVufDB8fHw%3D&auto=format&fit=crop&w=500&q=60',
-                    'https://images.unsplash.com/photo-1549849171-09f62448709e?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxleHBsb3JlLWZlZWR8M3x8fGVufDB8fHw%3D&auto=format&fit=crop&w=500&q=60',
-                    'https://images.unsplash.com/photo-1570997491915-47ade51fed9f?ixlib=rb-1.2.1&ixid=MXwxMjA3fDB8MHxleHBsb3JlLWZlZWR8MTJ8fHxlbnwwfHx8&auto=format&fit=crop&w=500&q=60'
-                    ]
+            userAnswers : {
```

Comment:
Thanks for changing the data structure to better mirror the actual data that we will need to display on the Style component